_sections/self-hosted-admin-android-studio.md

```diff
@@ -50,6 +50,13 @@ Finally, you will see a screen showing you that GitLive has been installed succe
 
 ![Successful Self Hosted Installation](/uploads/gsh-successful.jpg "Successful Self Hosted Installation"){:class="screenshot"}
 
+### Adding a Custom URL
+You can also add a custom URL that is accessible from the internet. If your remote URL is not accessible from the internet via HTTP or HTTPS, this custom URL will replace the remote URL with another one, so that our server can access the API.
```

Comment:
If your remote URL is not accessible from the internet via HTTP or HTTPS, you can add a custom URL that is accessible from the internet. This URL will be used for communication between your GitLab instance and our servers.